Lib/test/test_inspect.py

```diff
@@ -963,6 +964,33 @@ def test_nested_class_definition_inside_function(self):
         self.assertSourceEqual(mod2.cls213, 218, 222)
         self.assertSourceEqual(mod2.cls213().func219(), 220, 221)
 
+    def test_class_with_method_from_other_module(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            with open(os.path.join(tempdir, 'inspect_actual%spy' % os.extsep),
+                      'w', encoding='utf-8') as f:
+                f.write(textwrap.dedent("""\
```

Comment:
Nit: you don't need this backslash escape either. `textwrap.dedent` is smart enough to ignore an empty initial line.
```suggestion
                f.write(textwrap.dedent("""
```

Reply:
Thanks, I think I copied the backslash somewhere in other test cases.